txpipe/mapping/basic_maps.py

```diff
@@ -123,17 +124,20 @@ def finalize(self, comm=None):
             mask[count_g1>0] = True
             mask[count_g2>0] = True
 
-            #  Don't think we want to save these at the moment
-            del count_g1
-            del count_g2
+            # We only need to keep one of these as they should be
+            # the same.
+            counts_g[b] = count_g2.reshape(self.pixel_scheme.shape).flatten()
```

Comment:
Hum.... I'm slightly confused, why are g1 and g2 collected independently? Looks like it's possible to have bins with g1 contribution but no g2 from particular galaxies....

Reply:
Added